### ISSUE TITLE
MemoryError in sklearn/cluster/_k_means_elkan.pyx:235

### ISSUE THREAD

**Issue by amir20** [amir20/clashstats#552]
## Error in ClashLeaders

**MemoryError** in **sklearn/cluster/_k_means_elkan.pyx:235**

[View on Bugsnag](https://app.bugsnag.com/clash-leaders/clashleaders/errors/5bbad040048844001913acb8?event_id=5bbad040003044b8bd000000&i=gh&m=ci)

## Stacktrace

    sklearn/cluster/_k_means_elkan.pyx:235 - sklearn.cluster._k_means_elkan.k_means_elkan
    clashleaders/clustering/kmeans.py:23 - train_model
    clashleaders/batch/similar_clan.py:19 - compute_similar_clans
    clashleaders/scheduler.py:160 - main

[View full stacktrace](https://app.bugsnag.com/clash-leaders/clashleaders/errors/5bbad040048844001913acb8?event_id=5bbad040003044b8bd000000&i=gh&m=ci)

*Created automatically via Bugsnag*
